src/nico/src/nico/store.js

```diff
@@ -76,75 +80,68 @@ export default {
   actions: {
     setLanguage ({ commit }, language) {
       commit('setLangLoading', true)
-      commit('setLanguage', new languages[language](mars, () => commit('setLangLoading', false)))
+      commit(
+        'setLanguage',
+        new languages[language](() => commit('setLangLoading', false))
+      )
     },
 
-    run ({ state, commit, rootGetters, rootState, getters }) {
+    run ({ state, commit, rootGetters, rootState }) {
       commit('setClicks', state.clicks + 1)
       if (state.loading || state.langLoading) return
 
       commit('setView', 'game')
       commit('setRunning', false)
       commit('setErrors', [])
       commit('setLoading', false)
-      window.onerror = (message, source, lineno, colno, error) => {
-        commit('setErrors', [
-          state.language.convertWindowError({ message, source, lineno, colno, error }),
-        ])
-        commit('setRunning', false)
+
+      const { language } = state
+
+      if (!state.mars) {
```

Comment:
The problem is here; I have it only initialize `mars` once so that listeners don't get added multiple times for each `initMars` call, so the sprites getter that changes for each `run` call isn't updated in the context of `initMars`.